src/components/geonode.jsx

```diff
@@ -106,6 +106,21 @@ class GeoNodeViewer extends React.Component {
       errorOpen: false
     });
   }
+  _createLayerList() {
+    let layerList;
+    if(this._local) {
+      layerList = {
+        sources: [{title: this._local.title, url: this._local.url, type: 'WMS'}],
+        allowUserInput: true
+      };
+    } else {
+      layerList = {
+        sources: [{title: 'Local Geoserver', url: this.props.server+'/geoserver/wms', type: 'WMS'}],
```

Comment:
is there always a proxy available? My default geonode setup gives a 404 on http://localhost:8000/geoserver

Should this be 8080 instead as the default?